docs/demo/scripts/customise_interactions.py

```diff
@@ -206,194 +251,191 @@ def custom_handler_function(x):
     )
 )
 
-# Call function to trigger pulls
-my_fwtw.treat_water()
+# Run model for a timestep
+results = my_model.run(dates=[to_datetime("2000-01-01")])
 
 # Inspect new conditions
 print("Remaining reservoir storage: {0}".format(my_reservoir.tank.storage))
 print(
     "New service reservoir storage: {0}".format(my_fwtw.service_reservoir_tank.storage)
 )
 
+# View arc flows
+print(results[0])
+
 # %% [markdown]
 # We see some complaint from the `my_fwtw` about sludge (which are also explaining the small losses and why the new service reservoir volume is not quite 2), because it is anticipated to be connected to a sewer, but by inspecting the reservoir volumes we see that the `FWTW` has correctly pulled from the `Reservoir`.
 #
-# We can verify in a similar way that `my_demand` is able to pull from `my_reservoir`.
-
-# %%
-# Inspect initial conditions
-print("Initial storage: {0}".format(my_reservoir.tank.storage))
-print("Water supplied to demand: {0}".format(my_demand.total_received))
-
-# Call function to trigger pulls
-my_demand.create_demand()
-
-# Inspect new conditions
-print("Remaining reservoir storage: {0}".format(my_reservoir.tank.storage))
-print("Water supplied to demand: {0}".format(my_demand.total_received))
-
-# %% [markdown]
 # Again, a complaint that `my_demand` had nowhere to send its sewage, but we can see that it has successfully pulled
-# water from `my_reservoir`. Now we will update `my_reservoir` so that it denies pulls unless they have the tag `'FWTW'`.
+# water from `my_reservoir`. Now we will update `my_reservoir` so that it denies pulls unless they have the tag `FWTW`.
 
 # %% [markdown]
 # ### 2. Update reservoir handlers
-# This is a reasonably simple procedure, as we illustrated [above](#illustration).
-
+# As we illustrated [above](#illustration), we define our new handlers in a separate module, which we will call `custom_reservoir_handler.py`.
+#
+# <details>
```

Comment:
Same here: wrong format for the python code:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/e03b3291-2859-4ce3-8c13-482d3878447a">